pkg/master/controller.go

```diff
@@ -303,6 +304,7 @@ func (c *Controller) CreateOrUpdateMasterServiceIfNeeded(serviceName string, ser
 			// maintained by this code, not by the pod selector
 			Selector:        nil,
 			ClusterIP:       serviceIP.String(),
+			IPFamilyPolicy:  &singleStack,
```

Comment:
question for the future, I've added a method to the e2e framework that "identifies" the cluster IP family based in this service.
After these changes, is this assumption correct? The kubernetes.default IP family can be considered the cluster primary IP family?

Reply:
Best to worst answers
1. `Look at IPFamilies` <= That is the correct answer for any client. We have spent alot of time ensuring that IPFamilies will always be correct.
2. `test ClusterIP for ipv6`
3. `test ClusterIPs`